PubHub/PubHub.API/Domain/Entities/UserBook.cs

```diff
@@ -9,9 +9,9 @@ public class UserBook
         public DateTime AcquireDate { get; set; }
 
         #region Navs
-        public required Book Book { get; set; }
-        public required User User { get; set; }
-        public required AccessType AccessType { get; set; }
+        public required Book? Book { get; set; }
+        public required User? User { get; set; }
+        public required AccessType? AccessType { get; set; }
```

Comment:
Remove `requied` and use non-nullable type initialized with `null!`?

Reply:
Damn. Yea I see I forgot to correct this one (: